packages/venia-concept/src/components/Checkout/__tests__/braintreeDropin.spec.js

```diff
@@ -1,15 +1,10 @@
 import React from 'react';
 import TestRenderer from 'react-test-renderer';
+import waitForExpect from 'wait-for-expect';
 import dropin from 'braintree-web-drop-in';
-jest.mock('braintree-web-drop-in');
```

Comment:
Since I've defined a manual mock for this module, we no longer need to call `jest.mock()` for it.